### PR TITLE
web/user: Added Status section and custom Avatar

### PR DESCRIPTION
**Pull request by ironBatista**
#### Type Of Change 
- [ ] Small bug fix
- [X] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [X] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [X] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [X] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX)

### CODE REVIEW

File: web/portal/user/src/components/header/menuItems/MenuItems.styles.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    padding: '0px',
+    pointerEvents: 'none',
+    opacity: '0.8',
+  };
+});
+
+const Logo = styled('div')(() => {
+  return {
+    backgroundImage:
+      'url(https://user-ivozprovider.irontec.com/fso/webPortal/4-logo-ivoz.png)',
```
Comment (mmadariaga) [inline web/portal/user/src/components/header/menuItems/MenuItems.styles.tsx:49]: This should be dynamic

File: web/portal/user/src/components/header/menuItems/MenuItems.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+                overlap='circular'
+                anchorOrigin={{ vertical: 'bottom', horizontal: 'right' }}
+                variant='dot'
+                style={{ marginLeft: '5px', marginBottom: '2px' }}
+                color={status.registered ? 'success' : 'error'}
+              ></StyledBadge>
+            }
+          </p>
+          <p>{status.userAgent ? status.userAgent : 'No User Agent'}</p>
+          <p>sip:83012150624053601@192.168.5.160:5060</p>
```
Comment (mmadariaga) [inline web/portal/user/src/components/header/menuItems/MenuItems.tsx:50]: hardcoded text

**Review by mmadariaga — APPROVED** [review]

File: web/portal/user/src/components/header/menuItems/MenuItems.tsx (hunk 2 above)
Comment (ironBatista) [reply web/portal/user/src/components/header/menuItems/MenuItems.tsx:50]: Upps, sorry!

**Comment by ironArt3mis** [comment]
Rebase completed.

**Review by mmadariaga — APPROVED** [review]

**Comment by ironArt3mis** [comment]
Rebase completed.

**Pull request merged**